src/protocols/bitcoin/views/AccountDetailsTransactions.vue

```diff
@@ -1,17 +1,27 @@
 <template>
-  <div class="account-details-transactions">
-    <TransactionList
-      v-if="isOnline"
-      :transactions="transactions"
-      :loading="loading"
-      @load-more="loadMoreTransactions()"
-    />
-    <MessageOffline
-      v-else
-      class="offline-message"
-      :text="$t('modals.accountDetails.transactionsNotAvailable')"
-    />
-  </div>
+  <ion-page
+    class="account-details-transactions"
+  >
+    <ion-content
+      class="ion-padding"
+    >
```

Comment:
1. Please use PascalCase for component naming to maintain consistency.
2. I don't think splitting those lines is necessary.